engine/apps/schedules/migrations/0008_auto_20221201_0809.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.16 on 2022-12-01 08:09
+
+import pytz
+from django.db import migrations
+
+from common.timezones import is_valid_timezone
+
+
+def fix_bad_timezone_values(model):
+    def _fix_bad_timezone_values(apps, _schema_editor):
+        """
+            https://docs.djangoproject.com/en/4.1/topics/migrations/#data-migrations
+
+            We can't import the model directly as it may be a newer
+            version than this migration expects. We use the historical version.
+        """
+        Model = apps.get_model('schedules', model)
+        for obj in Model.objects.all():
+            if obj.time_zone and not is_valid_timezone(obj.time_zone):
+                obj.time_zone = pytz.UTC
```

Comment:
I'm not 100% sure if setting to UTC is the _best_ option but I suppose it's better than an invalid value that results in 500s?

Reply:
Right, agree.